resources/language/resource.language.en_gb/strings.po

```diff
@@ -773,34 +773,38 @@ msgid "Refresh watching activity"
 msgstr ""
 
 msgctxt "#30921"
+msgid "Remove watching activity"
```

Comment:
I am a bit confused about this new option.
When would the user remove the watching activity, and not disable the watching activity at the same time?
Because as long as the watching activity is enabled, it will be recreated by most listings.

So maybe we ought to remove it if "watching activity" is disabled and the file exists.

Reply:
Oh, I see that this is emptying everything? Do we need this?
What I proposed to add is an option to clean it up:
- Remove entries that are no longer offered by VRT NU
- Remove entries of which the resume position is not within expected range (e.g. not between 30 and -30)
- Remove entries that have no resume position set, and has watchLater == False

Reply:
No problem to remove this again.
I mainly added this because the add-on can't update resumepoints for programs like "Het goeie leven" (https://www.vrt.be/vrtnu/a-z/het-goeie-leven/) that don't have an `assetPath` in VRT NU Search API.
There are use cases where these programs are watched on VRT NU website and appear in the Continue and Watch later lists in the add-on, but can't be removed.

Reply:
How can they appear in the Continue and Watch later lists if they cannot be added to resumepoints?

Reply:
You can watch these episodes on VRT NU website...and this info is synced with the add-on.

Reply:
I have opened a separate pull request for this feature: https://github.com/pietje666/plugin.video.vrt.nu/pull/653